main.cpp

```diff
@@ -1,9 +1,22 @@
+#include "Brick.hpp"
 #include "Game.hpp"
 #include "Paddle.hpp"
 
 int main() {
     auto paddle = std::make_shared<Paddle>();
+
+    std::vector<std::shared_ptr<Brick>> bricks;
+
+    for (int col = 0; col < blocksCol; col++) {
+        for (int row = 0; row < blocksRow; row++) {
+            bricks.push_back(std::make_shared<Brick>((col * (blockWidth + 2)) + 3, (row * (blockHeight + 3)) + 100));
```

Comment:
I understand that 2 and 3 are spacing between each brick. But could you move that magic number to some constant?